staging/src/k8s.io/component-base/metrics/registry.go

```diff
@@ -188,6 +192,16 @@ func (kr *kubeRegistry) Register(c Registerable) error {
 	return nil
 }
 
+// Registerer exposes the underlying prometheus.Registerer
```

Comment:
is there anything consumers need to know about thread-safety, timing or restrictions on calling the Registerer / Gatherer?

Reply:
also, does exposing the underlying Registerer / Gatherer interfere with or circumvent any of the reasons we have our own intermediate layer (lazy registration, ensuring what gets registered conforms to expected names/patterns, etc)?

Reply:
@logicalhan